### ISSUE TITLE
Automatically purge old users

### ISSUE THREAD

**Issue by ZiggyMaes** [ZiggyMaes/Labos-ServersideApplications#24]
Users that haven't been active for over 6(?) months will be deleted.

**Comment by axd1967** [comment]
don't delete (purge) records, but hide them. (note: what you can do is to add a date field "deleted_date",then you also know when they were deleted.)

think of the impact of those "deleted" users on forum posts. e.g. the user name of a deleted user will no longer be clickable.

"6" becomes another site parameter (similar to admin's _"configure max message size"_)

**Comment by ZiggyMaes** [comment]
Okay, I agree with you there. I'll deactivate the users after 6 months, but then there's the issue of what to do when they want to come back. Letting them register again will introduce redundancy in the database, which I definitely do not want. I also just realized that e-mail support will introduce infrastructural issues (opening a new issue for this).

EDIT: #28

**Comment by axd1967** [comment]
good point about "deleted" users coming back. that's yet another (I'd say "extra") use case.

**Comment by ZiggyMaes** [comment]
So the assignment states that users should re-register after a period of inactivity. What if I do it like this: the user becomes 'inactive' after 6 months. When the user tries to login after being assigned this flag, they get a new form asking to update their information details.

Docent: OK.

**Comment by ZiggyMaes** [comment]
Code in place
